internal/mode/static/nginx/config/base_http_config_test.go

```diff
@@ -0,0 +1,48 @@
+package config
+
+import (
+	"strings"
+	"testing"
+
+	. "github.com/onsi/gomega"
+
+	"github.com/nginxinc/nginx-gateway-fabric/internal/mode/static/state/dataplane"
+)
+
+func TestExecuteBaseHttp(t *testing.T) {
```

Comment:
nit: looks like you could collapse these tests into one test with two cases since they are so similar.